independent-projects/bootstrap/maven-resolver/src/test/java/io/quarkus/bootstrap/resolver/maven/options/BootstrapMavenOptionsTest.java

```diff
@@ -0,0 +1,46 @@
+package io.quarkus.bootstrap.resolver.maven.options;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class BootstrapMavenOptionsTest {
+    private static String toRestore;
+
+    @BeforeAll
+    static void recordPreviousCmdLineProperty() {
+        toRestore = System.getProperty(BootstrapMavenOptions.QUARKUS_INTERNAL_MAVEN_CMD_LINE_ARGS);
+    }
+
+    @AfterAll
+    static void restorePreviousCmdLineProperty() {
+        if (toRestore != null) {
+            System.setProperty(BootstrapMavenOptions.QUARKUS_INTERNAL_MAVEN_CMD_LINE_ARGS, toRestore);
+        } else {
+            System.setProperty(BootstrapMavenOptions.QUARKUS_INTERNAL_MAVEN_CMD_LINE_ARGS, "");
```

Comment:
System.clearProperty(BootstrapMavenOptions.QUARKUS_INTERNAL_MAVEN_CMD_LINE_ARGS);

Reply:
I missed it. Was looking for `removeProperty`…